src/lumigo_opentelemetry/libs/general_utils.py

```diff
@@ -9,6 +9,15 @@
 
 from lumigo_opentelemetry import logger
 
+from opentelemetry.sdk.environment_variables import (
+    OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT,
+    OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT,
+)
+
+import os
```

Comment:
Can you move this "import" line up the file? Together with `typing` and `contextlib`?